src/app/client/src/app/modules/learn/components/learn-page/learn-page.component.ts

```diff
@@ -308,6 +309,12 @@ export class LearnPageComponent implements OnInit, OnDestroy {
     }
     this.playerService.playContent(event.data.metaData);
   }
+  viewAll(event) {
+    this.activatedRoute.data.subscribe(data => {
```

Comment:
dont take this from router.data, you can add this to config or hardcode the value here, as we know we are in learn page